File: Yoon/W5_2005.java
package Yoon;

import java.util.Scanner;

public class W5_2005 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int T;
        T = sc.nextInt();
        for (int t = 1; t <= T; t++) {
            int N = sc.nextInt();
            int[] pre = { 1 };
            int[] now;
            System.out.println("#" + t);
            for (int i = 1; i <= N; i++) {
                now = new int[i];
                for (int j = 0; j < i; j++) {
                    if (j == 0 || j == i - 1) {
                        now[j] = 1;
                    } else {
                        now[j] = pre[j - 1] + pre[j];
                    }
                    System.out.print(now[j] + " ");
                }
                System.out.println();
                pre = now;
            }

        }
    }
}

File: Yoon/W5_2005.py
T = int(input())
for t in range(1, T+1):
    N = int(input())
    pre = [1]
    print("#{0}".format(t))
    for i in range(1, N+1):
        now = []
        for j in range(i):
            if j == 0 or j == i-1:
                now.append(1)
            else:
                now.append(pre[j-1]+pre[j])
            print(now[j], end=" ")
        print()
        pre = now
